packages/model-viewer/src/features/scene-graph/image.ts

```diff
@@ -41,30 +41,28 @@ export class Image extends ThreeDOMElement implements ImageInterface {
     return this[$correlatedObjects]?.values().next().value as ThreeTexture;
   }
 
-  constructor(
-      onUpdate: () => void, texture: ThreeTexture|null,
-      gltfImage: GLTFImage|null) {
-    gltfImage = gltfImage ?? {
-      name: (texture && texture.image && texture.image.src) ?
-          texture.image.src.split('/').pop() :
-          'adhoc_image',
-      uri: (texture && texture.image && texture.image.src) ?
-          texture.image.src :
-          'adhoc_image' + adhocNum++
-    };
-    super(onUpdate, gltfImage, new Set<ThreeTexture>(texture ? [texture] : []));
+  get[$threeTextures](): Set<ThreeTexture> {
+    return this[$correlatedObjects] as Set<ThreeTexture>;
```

Comment:
Were you still planning to rename `$correlatedObjects`?

Reply:
I think the "correlated" name comes from the default correlation through the gltfMap from ThreeJS Loader: https://github.com/google/model-viewer/blob/8546f41450af57ee37fe7c00e3ef210138f6e3a0/packages/model-viewer/src/features/scene-graph/model.ts#L96

I'm not sure if the name should change.